jenkins-scripts/docker/ros_gz-install-test-job.bash

```diff
@@ -30,6 +30,10 @@ fi
 echo '# END SECTION'
 
 echo '# BEGIN SECTION: test gz_sim via ros2 launch'
+# Workaround until ros_gz defines a dependency on gz-sim*-cli. Avoid to use
+# apt --install-suggests since it installs the whole universe of packages
+VER=\$(dpkg -l | grep libgz-sim | grep ^ii | head -1 | awk '{ print \$3 '})
```

Comment:
Not sure why there's a `\$` instead of `$` and in the `awk '{ print \$3 '}`, should the `'` be after `}`?

Reply:
> Not sure why there's a \$ instead of $
An extra `\` is needed since we are in a bash `"""` block that is going to be consumed by bash, if we don't use that this will be translated to a bash variable instead of reaching the real execution as pure `$3`. 

> awk '{ print \$3 '}, should the ' be after }?

Umm this indeed seems a bug that bash passed just fine. I change it.